packages/artblocks/src/test-transactions.ts

```diff
@@ -0,0 +1,338 @@
+    contractAddress: '0x47e2df2723238f913741cc6b1963e76fdfd19b94',
+    tokenId: 429,
+    recipient: '0xbadb844dfb0ec559aaf3645b52ec3db0e0a3e124',
+  },
+}
+
+// https://etherscan.io/tx/0x5a0794d6f9bcaf260ef9a4f2553b7466c59fb4ce6be7a6301e2919fdd176db19
+export const PURCHASE_TO_DA_EXP_V4: TestParams<MintActionParams> = {
+  transaction: {
+    from: '0x1328a57ecfa4c1d88185899f81484de941114e46',
+    hash: '0x5a0794d6f9bcaf260ef9a4f2553b7466c59fb4ce6be7a6301e2919fdd176db19',
+    input:
+      '0x891407c00000000000000000000000001328a57ecfa4c1d88185899f81484de941114e4600000000000000000000000000000000000000000000000000000000000001c2',
+    to: '0x47e2df2723238f913741cc6b1963e76fdfd19b94',
+    value: '100000000000000000',
+    chainId: 1,
+  },
+  params: {
+    chainId: 1,
+    contractAddress: '0x47e2df2723238f913741cc6b1963e76fdfd19b94',
+    tokenId: 450,
+    recipient: '0x1328A57eCFA4c1d88185899F81484De941114E46',
+  },
+}
+
+// https://etherscan.io/tx/0xf3af8296d43e81e452ccae605978d9550409113b78f9ae448a2fa6ef6ef3a307
+export const INCOMPATIBLE_CONTRACT_PURCHASE: TestParams<MintActionParams> = {
+  transaction: {
+    chainId: 1,
+    from: '0x457ee5f723c7606c12a7264b52e285906f91eea6',
+    hash: '0xf3af8296d43e81e452ccae605978d9550409113b78f9ae448a2fa6ef6ef3a307',
+    input:
+      '0xefef39a10000000000000000000000000000000000000000000000000000000000000171',
+    to: '0x0bbb93c5d118d1dee49e96bcadc161403f4f8612',
+    value: '0',
+  },
+  params: {
+    chainId: 1,
+    contractAddress: '0x0bbb93c5d118d1dee49e96bcadc161403f4f8612',
+  },
+}
+
+// https://arbiscan.io/tx/0x085e59f8523be5da763dc44e9b7a13b4200dae9b846cbba559e4a1c033238287
+export const ARBITRUM_PURCHASE_TRANSACTION: TestParams<MintActionParams> = {
+  transaction: {
+    from: '0x5fe5f06e3471a1af24f9e8a1e866e16205d5df1a',
+    hash: '0x085e59f8523be5da763dc44e9b7a13b4200dae9b846cbba559e4a1c033238287',
+    input:
+      '0xae77c2370000000000000000000000000000000000000000000000000000000000000001000000000000000000000000d1d1222f6d3e4f64db1c025ecd0b314db8449ac4',
+    to: '0xe2bc24f74ed326ca4deb75753942731a566ebc83',
+    value: '2673000000000000',
+    chainId: 42161,
+  },
+  params: {
+    chainId: 42161,
+    contractAddress: '0xe2bc24f74ed326ca4deb75753942731a566ebc83',
+    tokenId: 1,
+    recipient: '0x5fe5f06e3471a1af24f9e8a1e866e16205d5df1a',
+  },
+}
+
+const chainTxnMap = {
+  mainnet: {
+    purchaseSetPriceV5: PURCHASE_SET_PRICE_V5,
+    purchaseToSetPriceV5: PURCHASE_TO_SET_PRICE_V5,
+    purchaseSetPriceMerkleV5: PURCHASE_SET_PRICE_MERKLE_V5,
+    purchaseToSetPriceMerkleV5: PURCHASE_TO_SET_PRICE_MERKLE_V5,
+    purchaseSetPriceV4: PURCHASE_SET_PRICE_V4,
+    purchaseToSetPriceV4: PURCHASE_TO_SET_PRICE_V4,
+    purchaseSetPriceERC20V4: PURCHASE_SET_PRICE_ERC20_V4,
+    purchaseMerkleV5: PURCHASE_MERKLE_V5,
+    purchaseToMerkleV5: PURCHASE_TO_MERKLE_V5,
+    purchaseDAExpV4: PURCHASE_DA_EXP_V4,
+    purchaseToDAExpV4: PURCHASE_TO_DA_EXP_V4,
+  },
+  Arbitrum: {
```

Comment:
Arbitrum ABIs match new V3 contract ABIs. This is merely testing that transactions on Arbitrum are passing as expected. Deeper coverage would be handled by overall ABI coverage.